homeassistant/components/bmw_connected_drive/translations/en.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
Unsure if en.json can't be generated from string.json

Reply:
It is generated from string.json using `python3 -m script.translations develop` to see some text in the dev environment. 
I just commited it with the rest as there was no  gitignore etc. but can remove if needed.